packages/badge/src/styled/index.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { IThemeBadgeProps } from '../interfaces';
+import { getRootBadgeStyles, getRootBadgeInteractiveStyles } from '../utils';
+
+
+export const BadgeRoot = styled.span<IThemeBadgeProps>`
+    display: inline-block;
+    padding: .25em .4em;
+    font-size: 75%;
+    font-weight: 700;
+    line-height: 1;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: .25rem;
+    ${({ theme, appearance = 'default', baseAppearance = 'default' }) => getRootBadgeStyles(theme, appearance, baseAppearance)}
```

Comment:
I think it would be better replace it in this way:
```
 ${getRootBadgeStyles}
 ${getRootBadgeInteractiveStyles}
```
This functions should accept `IThemeBadgeProps`.

Reply:
If you do as you suggest, then memoization will not work. Check it yourself 😊

Reply:
Better set `appearance = 'default', baseAppearance = 'default'` inside packages/badge/src/Badge.tsx
You will avoid logic duplication in this file.